chart/mq-queue-manager/templates/ibm-mq-queue-manager-installation.yaml

```diff
@@ -0,0 +1,19 @@
+kind: QueueManager
+apiVersion: mq.ibm.com/v1beta1
+metadata:
+  annotations:
+    com.ibm.mq/write-defaults-spec: 'false'
+  name: {{ .Values.queuemanagername }}
+  namespace: {{ .Values.queuemanagernamespace }}
+spec:
+  license:
+    accept: true
+    license: L-AXAF-JLZ53A
```

Comment:
this license, even if public, should be passed as value instead of hardcoding into the chart